src/lib/operators/join_sort_merge.cpp

```diff
@@ -132,6 +132,9 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractReadOnlyOperatorImpl {
   std::map<RowID, bool> _left_row_ids_emitted{};
   std::map<RowID, bool> _right_row_ids_emitted{};
 
+  std::vector<std::map<RowID, bool>> _left_row_ids_emitted_per_chunks;
```

Comment:
We are slowly moving to auto to stick: `auto _left_row_ids_emitted_per_chunks std::vector<std::map<RowID, bool>>{};

Reply:
If I do that the compiler is throwing the following error: `non-static data member declared with placeholder ‘auto’`. The reason is explained in: https://stackoverflow.com/questions/11302981/c11-declaring-non-static-data-members-as-auto .

Reply:
I moved the attributes to the `_perform_join` function. In there I used auto to stick. 🙃